var/spack/repos/builtin/packages/py-igor2/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIgor2(PythonPackage):
+    """igor2: interface for reading binary IGOR files."""
+
+    # pypi only has no sdist
+    homepage = "https://github.com/AFM-analysis/igor2"
+    url = "https://pypi.io/packages/py3/i/igor2/igor2-0.5.3-py3-none-any.whl"
```

Comment:
Could also use the GitHub tarball

Reply:
Ah, I'm able to reproduce this issue. Might be worth opening an issue with the devs and adding a comment to explain why GitHub sdist isn't used either.

Reply:
Nah, it's fine